docs/source/brain.rst

```diff
@@ -1759,6 +1766,89 @@ samples being less representative and closer samples being more representative.
    :alt: representativeness
    :align: center
 
+
+.. _brain-image-leaky-splits:
+
+Leaky Splits
+____________
+
+Despite our best efforts, duplicates and other forms of non-IID samples
+show up in our data. When these samples end up in different splits, this
+can have consequences when evaluating a model. It can often be easy to
+overestimate model capability due to this issue. FiftyOne Brain offers a way
+of identifying such cases in dataset splits.
+
+The leaks of a |Dataset| can be computed directly without the need
+for the predictions of a pre-trained model via the
+:meth:`compute_leaky_splits() <fiftyone.brain.compute_leaky_splits>`
+method:
+
+.. code-block:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.brain as fob
+
+    dataset = fo.load_dataset(...)
```

Comment:
Provide an actual example, like COCO. I found it sufficient to download the test split of COCO, untag and randomly split that into train, test, val, and perform the analysis on that